android/library/src/main/java/io/logto/android/Logto.kt

```diff
@@ -1,19 +1,82 @@
 package io.logto.android
 
+import android.app.Application
 import android.content.Context
 import io.logto.android.authflow.webview.WebViewAuthFlow
+import io.logto.android.callback.AuthenticationCallback
+import io.logto.android.config.LogtoConfig
 import io.logto.android.model.Credential
 import io.logto.android.storage.CredentialStorage
 
-class Logto {
-    companion object {
-        private var credentialStorage: CredentialStorage? = null
+class Logto private constructor() {
+
+    private lateinit var logtoConfig: LogtoConfig
+
+    private lateinit var application: Application
+
+    private var inited: Boolean = false
+
+    private var useStorage: Boolean = false
+
+    private var credentialStorage: CredentialStorage? = null
+
+    private var _credential: Credential? = null
```

Comment:
if we don't prepend `_` for private members by default, then all the naming should follow this convention.